src/js/components/stock-movement-wizard/PickPage.jsx

```diff
@@ -268,7 +268,7 @@ class PickPage extends Component {
   }
 
   fetchPickPageItems() {
-    apiClient.get(`/openboxes/api/stockMovements/${this.state.values.stockMovementId}?stepNumber=4`)
+    apiClient.post(`/openboxes/api/stockMovements/${this.state.values.stockMovementId}/updateAdjustedItems`)
```

Comment:
the stockAdjustments endpoint will return the list of adjusted items, you can use it and send it to the updateAdjustedItems and make the update only for this stock movement items